lightning/src/ln/channelmanager.rs

```diff
@@ -5276,6 +5276,16 @@ impl<'a, Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref>
 				None => continue,
 			}
 		}
+		if forward_htlcs_count > 0 {
+			// If we have pending HTLCs to forward, assume we either dropped a
+			// `PendingHTLCsForwardable` or the user received it but never processed it as they
+			// shut down before the timer hit. Either way, set the time_forwardable to a small
+			// constant as enough time has likely passed that we should simply handle the forwards
+			// now, or at least after the user gets a chance to reconnect to our peers.
+			pending_events_read.push(events::Event::PendingHTLCsForwardable {
+				time_forwardable: Duration::from_secs(2),
```

Comment:
Do we still expect the user to wait between "now + 2 time forwardable" and "now + 5 * time forwardable" following `PendingHTLCsForwardable` doc ? Not sure if it makes sense to wait at least 5 * after reload as all pending HTLCs have been normalized in the same period due to the serialization. If yes, maybe doc could suggest that.

Reply:
Yea, we haven't changed the docs here, but we would have to do something other than *just* change the docs - a user cannot determine just by looking at a `PendingHTLCsForwardable` whether it was created during startup or at runtime, so I dunno if we want to put that complexity on the user. In the future we can/should just do the randomization ourselves, IMO, now that we have a generic random value interface, but that's a bigger change.